skunkworks_crow/src/main/java/org/odk/share/views/ui/bluetooth/BtReceiverActivity.java

```diff
@@ -225,4 +234,22 @@ protected void onDestroy() {
         super.onDestroy();
         unregisterReceiver(bluetoothReceiver);
     }
+
+    @Override
+    public void onBackPressed() {
```

Comment:
In receiver we should not ask like 'To disable bluetooth', rather we should ask `Do you want to stop the receiving` or any other meaningful message, this way we'll close the connection and clean up the half downloaded forms/files which were in progress to avoid any mismatch in next transaction.

Reply:
Sure, will fix soon.